project/Dependencies.scala

```diff
@@ -110,16 +126,16 @@ object Dependencies {
 
   val DiscoveryAwsApi = Seq(
     libraryDependencies ++=
-      DependencyGroups.AkkaActor ++ Seq(
-        "com.amazonaws" % "aws-java-sdk-ec2" % "1.11.292",
-        "com.amazonaws" % "aws-java-sdk-ecs" % "1.11.292")
+      DependencyGroups.AkkaActor ++
+      DependencyGroups.AwsJavaSdkEc2Ecs // aws depends on insecure version
   )
 
   val DiscoveryAwsApiAsync = Seq(
     libraryDependencies ++=
-      DependencyGroups.AkkaActor ++ Seq(
-        "software.amazon.awssdk" % "ecs" % "2.0.0-preview-9",
-        "org.scala-lang.modules" %% "scala-java8-compat" % "0.8.0")
+      DependencyGroups.AkkaActor ++
+      DependencyGroups.Aws2Ecs ++ Seq(
+        "org.scala-lang.modules" %% "scala-java8-compat" % "0.8.0"
```

Comment:
The compat dependency should be removed
It was fixed in master in https://github.com/akka/akka-management/pull/399

Reply:
Done